svea_vision/README.md

```diff
@@ -1,11 +1,91 @@
 # svea_vision
+This package contains the vision system for SVEA. It contains nodes for object detection, pose estimation, aruco markers and sidewalk segmentation. 
 
 ## Installation
 
-Please refer to [svea](https://github.com/KTH-SML/svea) on how to install and
-run SVEA software.
+Please refer to [svea](https://github.com/KTH-SML/svea) on how to install and run SVEA software.
 
-## Usage
+## ROS Nodes
```

Comment:
Prefer to use docstrings for details, we can have oneline descriptions of each node here.